processing/src/main/java/org/apache/druid/segment/transform/Transformer.java

```diff
@@ -93,6 +93,31 @@ public InputRow transform(@Nullable final InputRow row)
     return transformedRow;
   }
 
+  @Nullable
+  public InputRowListPlusJson transform(@Nullable final InputRowListPlusJson row)
+  {
+    if (row == null) {
+      return null;
+    }
+
+    final InputRowListPlusJson transformedRow;
+
+    if (transforms.isEmpty()) {
+      transformedRow = row;
+    } else {
+      transformedRow = InputRowListPlusJson.of(new TransformedInputRow(row.getInputRow(), transforms), row.getRaw());
```

Comment:
should this be transforming the list of `InputRow` to `TransformedInputRow`?

Reply:
👍 fixed.